website/search/elastic_search.py

```diff
@@ -467,7 +467,9 @@ def search_contributor(query, page=0, size=10, exclude=None, current_user=None):
         # TODO: use utils.serialize_user
         user = User.load(doc['id'])
 
-        if current_user:
+        if current_user == user:
```

Comment:
`==` comparison of modm objects is a little finicky. It's safer to check IDs (current_user._id == user._id).